ntbs-service/Pages/Notifications/Edit/Items/ManualTestResult.cshtml.cs

```diff
@@ -181,5 +190,12 @@ public async Task<JsonResult> OnGetFilteredSampleTypesForManualTestType(int valu
                         .Select(result => OptionValue.FromEnum(result))
                 });
         }
+
+        public ContentResult OnPostValidateManualTestResultProperty([FromBody]InputValidationModel validationData)
+        {
+            // This is a workaround for the fact we need to use a validate-input component on our result field
+            // but we cannot validate it here as we do not know the test type
```

Comment:
The conditional-select-wrapper expects to have a validate input inside it. It is built so that a change event in the select will bubble up through that to the wrapper.

Reply:
Ok, I'm not terribly happy about this, but probably not worth changing